AnnoyancesFilter/Other/sections/annoyances.txt

```diff
@@ -3229,3 +3229,6 @@ adguard.info,adguard.com,adguard.app##.hello_from_adguard_ext
 !+ PLATFORM(ios) NOT_OPTIMIZED
 adguard.info,adguard.com,adguard.app#$#.hello_from_adguard_advanced_protection_ios { display: none !important; }
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/184349
+report.az##main.content:nth-child(3) > section.news-inner-page.pt-20:first-child > div.container > div.row > div.col-lg-8.news-col:first-child > div.news-wrapper > div.selected-news:first-child > div.editor-body:nth-child(8) > div.subs-in-social.subs-telegram:last-child
+report.az##main.content:nth-child(3) > section.news-inner-page.pt-20:first-child > div.container > div.row > div.col-lg-8.news-col:first-child > div.news-wrapper > div.selected-news:first-child > div.editor-body:nth-child(8) > div.subs-in-social.subs-instagram:last-child
```

Comment:
We don't usually use a rule like this formation.
Please take a look the repo softly.